server/crawlers/twitterCrawler.js

```diff
@@ -7,7 +7,7 @@ async function twitterCrawler(companyName) {
     bearer_token: process.env.Twitter_bearer_token,
   });
 
-  let tweets = await client.get("tweets/search/30day/dev", {
+  let tweets = await client.get("tweets/search/30day/hatchways", {
```

Comment:
What's this for?

Reply:
I believe @grogsy created another api key because the original one hit our limit, this might be what he called it?